lib/libzpool/uzfs_mgmt.c

```diff
@@ -323,6 +326,8 @@ uzfs_open_dataset_init(spa_t *spa, const char *ds_name, zvol_state_t **z)
 disown_free:
 		dmu_objset_disown(zv->zv_objset, zv);
 free_ret:
+		if (spa != NULL)
```

Comment:
I think if we are here, we have spa non-null, so to me check is un-necessary.

Reply:
right. thanks! I will fix it